applications/nrf_desktop/doc/hid_forward.rst

```diff
@@ -119,18 +119,18 @@ Forwarding HID output reports
 =============================
 
 When the |hid_forward| receives a ``hid_report_event`` that contains an output report from a :ref:`nrf_desktop_usb_state`, it tries to forward the output report.
-The HID output report is forwarded to all of the Bluetooth connected peripherals that forward the HID data to HID subscriber that is source of the HID output report.
+The HID output report is forwarded to all of the Bluetooth connected peripherals that forward the HID data to the HID subscriber that is source of the HID output report.
```

Comment:
Please see line 109 in this document. It seems there is a missing word after "Up to" ?

> _Up to :kconfig:`CONFIG_DESKTOP_HID_FORWARD_MAX_ENQUEUED_REPORTS` reports can be enqueued at a time for each report type and for each connected peripheral._

Reply:
```
Up to the number of reports speficied in :kconfig:CONFIG_DESKTOP_HID_FORWARD_MAX_ENQUEUED_REPORTS can be enqueued at a time for each report type and for each connected peripheral.
```

Reply:
Alternatively
```
A number of reports up to the number specified in :kconfig:`CONFIG_DESKTOP_HID_FORWARD_MAX_ENQUEUED_REPORTS` can be enqueued at a time for each report type and for each connected peripheral.
```
Not sure which reads better.

Reply:
I will go with the first suggestion, because it is closest to the original but all are okay IMO.

Reply:
Thanks for fixing the typo :D